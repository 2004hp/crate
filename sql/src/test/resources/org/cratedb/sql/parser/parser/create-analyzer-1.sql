create analyzer "my_analyzer" (
    tokenizer (
        bla='blubb'
    )
)