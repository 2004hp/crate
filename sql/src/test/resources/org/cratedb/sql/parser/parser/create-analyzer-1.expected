org.cratedb.sql.parser.parser.CreateAnalyzerNode@796203a7
name: my_analyzer
statementType: CREATE ANALYZER
	
	org.cratedb.sql.parser.parser.AnalyzerElements@7aaae6f

	Tokenizer:
		
		org.cratedb.sql.parser.parser.TokenizerNode@56a1fd57
		{bla=value: blubb
		type: CHAR(5) NOT NULL
		}
	TokenFilters:
		
		org.cratedb.sql.parser.parser.TokenFilterList@24ff8730

	CharFilters:
		
		org.cratedb.sql.parser.parser.CharFilterList@404734a
